### DISCUSSION TITLE
截图(screenshot video)受亮度/伽马调整影响

### DISCUSSION THREAD

**Discussion by yeezylife** [hooke007/mpv_PlayKit#127] [Q&A 问答] [answered]
有时候房间很亮,我看比较暗的视频会把亮度调高点看,然后我截图的时候希望截原始画面,原始亮度...
但是我发现截出来的图片会受到亮度/伽马等调整的影响,这样我需要恢复原始亮度=>截图=>调亮一点继续看...很蠢...
有没有方法可以让截图(screenshot video)不受亮度/伽马调整影响呢

**Comment by hooke007** [comment, marked as answer]
正常不应该是调节显示设备亮度吗。。。
均衡器也是整个渲染的一环，这个问题相当于截图前禁用滤镜/着色器，截图后再自动切回来（

**Reply by yeezylife** [reply]
我想也是,应该就是这么设计的.  我坐窗边啦 太阳大的时候房间很亮 显示器来回调应该更麻烦... 
这个应该算特殊情况.

**Reply by hooke007** [reply]
问题是，你这是错误操作。准确讲mpv的“亮度”调节的是明度，显示器调节的才是真亮度。这两概念的区别是：明度拉到顶或者底，画面直接失去明暗关系（已经不是丢细节了，失真到离谱）。调节亮度不会丢失画面的明暗关系。

**Reply by yeezylife** [reply]
我调整的范围很小 看特别暗的电影,我会调一下伽马..亮度动的更少
当然啦显示器调整到(当时)合适的亮度当然是最好的...

**Reply by hooke007** [reply]
如果你的外部显示器不是奇葩的协议的话，是可以用 https://github.com/xanderfrangos/twinkle-tray 调节亮度的

**Reply by yeezylife** [reply]
哇这个好

**Reply by yeezylife** [reply]
还能分时间段设置亮度!!!开心! 我最喜欢hooke007了

**Reply by yeezylife** [reply]
> 画面直接失去明暗关系。

关于这个准确的画面明暗关系,好像mpv默认的BT1886本身就会破坏明暗关系,会把暗处变亮(lift the shadows)

**Reply by hooke007** [reply]
你在说icc还是什么，no-config下和所有播放器都一样，不存在默认破坏明暗关系

**Reply by yeezylife** [reply]
https://github.com/mpv-player/mpv/issues/8082

我在这个issue里看人家讨论提到的,可能我理解有问题

**Reply by hooke007** [reply]
这不就是我说的icc吗
